File: Tugas coding modul 5/coding c/PRAK501-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d) {
    int hasil;

    if (a > b && a > c && a > d) {
        hasil = a;
    } else if (b > a && b > c && b > d) {
        hasil = b;
    } else if (c > a && c > b && c > d) {
        hasil = c;
    } else if (d > a && d > b && d > c) {
        hasil = d;
    }
    
    return hasil;
}
int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);

    return 0;
}

File: Tugas coding modul 5/coding py/PRAK501-2310817120005-Noviana_Nur_Aisyah.py
def MaxBilangan(a, b, c, d):
    if a > b and a > c and a > d:
        hasil = a
    elif b > a and b > c and b > d:
        hasil = b
    elif c > a and c > b and c > d:
        hasil = c
    elif d > a and d > b and d > c:
        hasil = d

    return hasil

def main():
    a, b, c, d = map(int, input().split())
    hasil = MaxBilangan(a, b, c, d)
    print(hasil)

main()